bin/commands/info.js

```diff
@@ -4,33 +4,28 @@ const request = require('request');
 const config = require("../helpers/config"),
   logger = require("../helpers/logger").winstonLogger,
   Constants = require("../helpers/constants"),
-  util = require("../helpers/util");
+  utils = require("../helpers/utils");
```

Comment:
Why the pluralization?

Reply:
Because `util` is inbuilt node.js module. Hence changed it to `utils` for user defined module.